frontend/src/app/revisions.test.tsx

```diff
@@ -128,6 +167,66 @@ describe('Revisions tests', () => {
         expect(inst.onHeadingToggle()).toMatchSnapshot();
     });
 
+    it('test previewDoc function', () => {
+        const wrapper = renderer.create(<Revisions {...props} />);
+        const inst = wrapper.getInstance();
+        expect(inst.previewDoc()).toMatchSnapshot();
+    });
+
+    it('test saveMetadata function', () => {
+        const wrapper = renderer.create(<Revisions {...props} />);
+        const inst = wrapper.getInstance();
+        expect(inst.saveMetadata()).toMatchSnapshot();
+    });
+
+    it('test onChangeProduct function', () => {
+        const wrapper = renderer.create(<Revisions {...props} />);
+        const inst = wrapper.getInstance();
+        expect(inst.onChangeProduct()).toMatchSnapshot();
+    });
+
+    it('test onChangeUsecase function', () => {
+        const wrapper = renderer.create(<Revisions {...props} />);
+        const inst = wrapper.getInstance();
+        expect(inst.onChangeUsecase()).toMatchSnapshot();
+    });
+
+    it('test handleURLInput function', () => {
+        const wrapper = renderer.create(<Revisions {...props} />);
+        const inst = wrapper.getInstance();
+        expect(inst.handleURLInput()).toMatchSnapshot();
+    });
+
+    // it('test moduleUrlExist function', () => {
```

Comment:
if we don't need this test let's also remove it instead of commenting it out.

Reply:
done